tests/console/shar.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This will test the shar (SHELL ARCHIVE) tool
+# Maintainer: Dominik Heidler <dominik@heidler.eu>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+
+sub run {
+    my $self = shift;
+    select_console 'user-console';
```

Comment:
perhaps the serial terminal is a better choice here?

Reply:
As soon as serial terminal output is displayed in a usable way in the openQA webui, I will start using it.